docs/SPEC.md

```diff
@@ -3400,15 +3400,6 @@ Example: `from(bucket: "telegraf") |> filter(fn:(r) => r._measurement == "mem" a
 The function `toFloat` is defined as `toFloat = (tables=<-) => tables |> map(fn:(r) => float(v:r._value))`.
 If you need to convert other columns use the `map` function directly with the `float` function.
 
-##### toDuration
-
-Convert a value to a duration.
-
-Example: `from(bucket: "telegraf") |> filter(fn:(r) => r._measurement == "mem" and r._field == "used") |> toDuration()`
-
-The function `toDuration` is defined as `toDuration = (tables=<-) => tables |> map(fn:(r) => duration(v:r._value))`.
-If you need to convert other columns use the `map` function directly with the `duration` function.
-
```

Comment:
Why is this being removed?

Reply:
tables don't have a duration type.  we can add it back in if we wanted to extend tables to include duration columns.